tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -282,3 +284,45 @@ def test_streaming_non_streaming_generate_same_tokens(pipeline, prompt):
         tokens.append(g.generations[0].text)
     output_2 = "".join(tokens)
     assert output_1 == output_2
+
+
+def test_edge_cases(pipeline, prompt):
+    output = pipeline(prompt=prompt, max_length=1, output_scores=True)
```

Comment:
please add small comments to each test case to flag which type of behavior it tests